egs/chime5/s5b/local/chain/tuning/run_tdnn_1b.sh

```diff
@@ -0,0 +1,290 @@
+#!/bin/bash
+
+# 7q is as 7p but a modified topology with resnet-style skip connections, more layers,
```

Comment:
The comment here is outdated, it looks like it's from Switchboard.   You need to make sure these headers are updated, with relevant description and results.